### PR TITLE
docs: README.md and resource improvements

### PR DESCRIPTION
**Pull request by Lakshamana**
* Add Software Requirements section
* Add Contributing section
* Add Usage section
* Add demo and resources

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,46 +6,91 @@ schedgen
...
 avatar_x = 80  # based off of the center of the rectangle
+avatar_margin = 9  # margin around avatar
 
 [schedgen.streamers]
-'vinnydays' = { avatar='./vinnydays.png', service='twitch.tv' }
-'ponzuzuju' = { avatar='./ponzuzuju.png', service='twitch.tv' }
-'gamerdeesquerda' = { avatar='./gamerdeesquerda.png', service='twitch.tv' }
-'0froggy' = { avatar='./0froggy.png', service='twitch.tv' }
-'historiapublica' = { avatar='./historiapublica.png', service='twitch.tv' }
+'jo3star' = { avatar='./demo/joestar.png', service='twitch.tv' }
```
Comment (tarcisioe) [inline README.md:65]: Ugh.

File: README.md [hunk 2]
```diff
@@ -6,46 +6,91 @@ schedgen
...
+- Clone your fork into local machine
+- Run `poetry install` to install packages
+- Run `poetry run pre-commit install` to install commit hooks
+- Run `poetry shell` to get a poetry shell, in which you may run:
+  - `summon test` for running tests
+  - `summon format` for code formating
+  - `summon lint` for code linting
+  - `schedgen` for actually testing the tool
+- Commit your changes (squashing before merging is quite a good idea =))
+- Open a PR and request reviews
```
Comment (tarcisioe) [inline README.md:93]:
```suggestion
- Open a PR and request reviews

Contributors
------------

- @tarcisioe (barely tolerates the Jojo reference)
- @Lakshamana (responsible for Jojo reference)
```

File: README.md [hunk 3]
```diff
@@ -6,46 +6,91 @@ schedgen
...
+schedgen [OPTION]... WEEKDAY STREAMER...
+```
+
+Generates schedule announcement for streamers (check [example](./example.png)).
+
+Where OPTION may be one of following options:
+
+| OPTION        | Description                                             |
+|---------------|---------------------------------------------------------|
+| --background  | Background image relative path                          |
```
Comment (tarcisioe) [inline README.md]: Just "path", it could be relative or absolute, both are fine.

File: README.md [hunk 4]
```diff
@@ -6,46 +6,91 @@ schedgen
 
 This is a schedule image generator for streamer teams!
 
-How to use
-----------
+Software Requirements
+---------------------
+- [poetry](https://python-poetry.org): package manager (it should install summon as dev dependency)
```
Comment (tarcisioe) [inline README.md]: You can link to https://github.com/tarcisioe/summon-python here.

File: README.md [hunk 5]
```diff
@@ -6,46 +6,91 @@ schedgen
...
+- Run `fc-match <FontName>` to get OS-hosted font file (change `<FontName>` for a font name of your choice)
```
Comment (tarcisioe) [inline README.md]: I believe fontconfig is pretty much Linux-only.

File: README.md [hunk 6]
```diff
@@ -6,46 +6,91 @@ schedgen
...
 
 [schedgen.entry_style]
 stroke_width = 7
-max_height = 123
-min_spacing = 7
-width = 285
+max_height = 120
+min_spacing = 7  # min_spacing
+width = 285  # rectangle's width
 stroke_color = { r=238, g=17, b=75 }
```
Comment (tarcisioe) [inline README.md]: Maybe user something with better contrast here.

File: README.md [hunk 7]
```diff
@@ -6,46 +6,91 @@ schedgen
...
 schedule_y = 87  # from the top
+weekday_y = 32  # from the top
 
 [schedgen.entry_style]
 stroke_width = 7
-max_height = 123
-min_spacing = 7
-width = 285
+max_height = 120
+min_spacing = 7  # min_spacing
```
Comment (tarcisioe) [inline README.md]: The comment is just the name of the variable.

**Review by tarcisioe — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Lakshamana) [reply README.md:65]: =)

File: README.md (hunk 7 above)
Comment (Lakshamana) [reply README.md]: oh, sorry. I forgot this

File: README.md (hunk 3 above)
Comment (Lakshamana) [reply README.md]: ok

File: README.md (hunk 5 above)
Comment (Lakshamana) [reply README.md]: Perhaps mac users will get this too. I cannot ensure it for Windows users tho, so I'll just remove this line

File: README.md (hunk 2 above)
Comment (Lakshamana) [reply README.md:93]: added

**Review by tarcisioe — APPROVED** [review]

**Pull request merged**